scripts/enemy.gd

```diff
@@ -0,0 +1,27 @@
+extends KinematicBody2D
+class_name Enemy
+
+export var speed := 100.0
```

Comment:
Line 4 and 6 start go give a type but dont. Its hard to tell if its a float double or int

Reply:
done